oc_jb_workaround/src/com/owncloud/android/workaround/accounts/AccountAuthenticatorService.java

```diff
@@ -60,8 +60,7 @@ public AccountAuthenticator(Context context) {
         @Override
         public Bundle addAccount(AccountAuthenticatorResponse response,
                 String accountType, String authTokenType,
-                String[] requiredFeatures, Bundle options)
-                throws NetworkErrorException {
```

Comment:
Not sure if we should delete this exception, is being used if the authenticator could not honor the request due to a network error.
Info: https://developer.android.com/reference/android/accounts/AbstractAccountAuthenticator.html

Reply:
Well but if you look into the code there is only an intent reequest, we don't actually call a network method in there.
Also java does not care if an overriding function throws a certain exception or not. If you declare a method in an abstract using certain exceptions, you may only throw a subset of those exceptions. That includes throwing no exceptions.